node/hack/repair_eth/repair_eth.go

```diff
@@ -48,6 +48,7 @@ var etherscanAPIMap = map[vaa.ChainID]string{
 	vaa.ChainIDBase:     "https://api.basescan.org",
 	vaa.ChainIDScroll:   "https://api.scrollscan.com",
 	vaa.ChainIDBlast:    "https://api.blastscan.io",
+	vaa.ChainIDXLayer:   "", // TODO: Does X Layer have an etherscan API endpoint?
```

Comment:
official explorer is [here](https://www.okx.com/web3/explorer/xlayer), with another one [here](https://www.oklink.com/xlayer) that has [api plans](https://www.oklink.com/api-plans) I think.

Reply:
That requires a log in, which I don't think we want to do. Will address this when I hear back from OKX.